scripts/cover/coverGcov.sh

```diff
@@ -1,50 +1,46 @@
 #!/usr/bin/env bash
 set -e
-
 # set HOOT_HOME if not set
 if [ -z "$HOOT_HOME" ]; then
     HOOT_HOME=~/hoot
 fi
-
 # generate gcov files from gcda files
 cd $HOOT_HOME
-
+# create the gcov output folder
+if [ ! -d gcov ]; then
+    mkdir gcov
+else
+    rm -f gcov/*
+fi
 # hoot-core coverage
-mkdir gcovCore
-cd gcovCore
-gcov ../hoot-core/tmp/debug/*.gcda
+pushd hoot-core
+gcov -o $HOOT_HOME/hoot-core/tmp/debug/ $HOOT_HOME/hoot-core/tmp/debug/*.gcda
 # fix path
-sed -i 's/Source:src/Source:hoot-core\/src/g' *.gcov
-sed -i 's/Source:..\//Source:/g' *.gcov
-mv *.gcov ../
-cd ..
-
+sed -i "s|Source:src|Source:$HOOT_HOME/hoot-core/src|g" *.gcov
```

Comment:
The vertical pipe `|` can be used in `sed` regex so that the `/` in `$HOOT_HOME` isn't interpreted as a delimiter.

Reply:
Clean.  I like it.